forge/src/main/java/com/gregtechceu/gtceu/api/registry/forge/GTRegistriesImpl.java

```diff
@@ -29,6 +30,7 @@ public static <V, T extends V> T register(Registry<V> registry, ResourceLocation
         if (registryKey == RECIPE_TYPE_REGISTRY) {
             ForgeRegistries.RECIPE_TYPES.register(name, (RecipeType<?>) value);
         } else if (registryKey == Registry.RECIPE_SERIALIZER_REGISTRY) {
+            //have to register multiple keys to same value here, sorry forge
```

Comment:
Does forge emit a warning here? If so, we should log a message explaining why it should be ignored by users.

Reply:
it doesn't (it threw an exception instead & crashed), so i just... made new serializer instances for each type. can't be fixed otherwise, blame forge.